### PR TITLE
CLDR-14615 Modernize by fixing UI glitches

### PR DESCRIPTION
**Pull request by btangmu**
-New GearMenu.vue and new special #gear, not attached to navbar yet

-Fix bug in cldrMail.js: myLoad should be cldrLoad.myLoad

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14615
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/cldr-apps/js/src/views/GearMenu.vue [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    <li v-if="isTC">
+      <ul>
+        <li><a href="#error_subtypes">Error Subtypes</a></li>
+      </ul>
+    </li>
+  </ul>
+</template>
+
+<script>
+import * as cldrStatus from "../../../src/main/webapp/js/esm/cldrStatus.js";
```
Comment (srl295) [inline tools/cldr-apps/js/src/views/GearMenu.vue:115]: not needed. It's already `$cldrOpts.cldrStatus`

File: tools/cldr-apps/js/src/views/GearMenu.vue [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    };
+  },
+
+  created() {
+    this.initializeData();
+  },
+
+  methods: {
+    initializeData() {
+      const perm = cldrStatus.getPermissions();
```
Comment (srl295) [inline tools/cldr-apps/js/src/views/GearMenu.vue:137]: already there as `this.$cldrOpts.cldrStatus` etc.

File: tools/cldr-apps/src/main/webapp/js/esm/cldrText.js [hunk 3]
```diff
@@ -436,6 +436,7 @@ const strings = {
   special_flagged: "Flagged Items",
   special_forum: "Forum Posts",
   special_forum_participation: "Forum Participation",
+  special_gear: "⚙",
```
Comment (srl295) [inline tools/cldr-apps/src/main/webapp/js/esm/cldrText.js:439]: 💯

**Review by srl295 — APPROVED** [review]

File: tools/cldr-apps/js/src/views/GearMenu.vue (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/js/src/views/GearMenu.vue:115]: I kind of like an explicit "import" statement anyway -- more modular

**Review by srl295 — APPROVED** [review]

**Pull request merged**